SCMS.Services.Api/Brokers/Storages/StorageBroker.Occupation.References.cs

```diff
@@ -0,0 +1,27 @@
+﻿// -----------------------------------------------------------------------
+// Copyright (c) Signature Chess Club & MumsWhoCode. All rights reserved.
+// -----------------------------------------------------------------------
+
+using Microsoft.EntityFrameworkCore;
+using SCMS.Services.Api.Models.Foundations.Occupations;
+
+namespace SCMS.Services.Api.Brokers.Storages
+{
+    public partial class StorageBroker
+    {
+        public void SetOccupationReferences(ModelBuilder modelBuilder)
+        {
```

Comment:
The Guardian and Occupations relation is missing here.

Reply:
Will it end with .OnDelete(DeleteBehavior.NoAction); ?